static/app/views/monitors/components/monitorForm.tsx

```diff
@@ -382,7 +382,7 @@ function MonitorForm({
               />
               <NumberField
                 name="config.max_runtime"
-                placeholder={`Defaults to ${DEFAULT_MAX_RUNTIME} minutes`}
+                placeholder={t('Defaults to %s minutes', DEFAULT_MAX_RUNTIME)}
```

Comment:
use `tn()` and watch out for `minute` vs `minutes`

Reply:
yeah that's a good point